BibBuddy/app/src/main/java/de/bibbuddy/IsbnRetriever.java

```diff
@@ -37,18 +37,31 @@ private static Document loadXmlFromString(String xml) throws Exception {
     return builder.parse(inputSource);
   }
 
+  private String getField(Document xmlMetadata, String fieldName) {
+    String value = "";
+
+    try {
+      value = xmlMetadata.getElementsByTagName(fieldName).item(0).getTextContent();
+    } catch (Exception e) {
+      // exception for publication year, which needs an int
+      if (fieldName == "dcterms:issued") {
+        value = "0";
+      }
+      System.out.println("Value of field '" + fieldName + "' could not be extracted.");
```

Comment:
```suggestion
      e.printStackTrace();
```
Would be better and more consistent for catching error in logs than creating custom system prints.

Reply:
✓